bc/contracts/Votes.sol

```diff
@@ -6,26 +6,46 @@ contract Votes is Ownable {
 
     uint voteFee = 0.001 ether;
 
-     struct PostVote {
+     struct Vote {
          bytes32 vote_hash;
-         bytes32 post_hash;
+         bytes32 target_hash;
```

Comment:
Let's make some short call in future for general contracts design and mini-audit review 😄

Reply:
These implementations that I have right now are probably pretty lame, now I'll design the economy (that's the next thing I'm going after) and with that I'll start implementing those *serious* contracts.

Right now I know only some of the possible vulnerabilities in Solidity, but I'm going to review them a few times while implementing the new contracts.